frontend/src/components/uam-configure-group/uam-configure-group.tsx

```diff
@@ -45,10 +45,12 @@ const UAMConfigureGroup: FC = () => {
     usePagination({ queryName: 'users' });
 
   const handleCreateOrEdit = (): void => {
+    const { name } = getValues();
```

Comment:
`handleSubmit` passes form formValues on submit action
```suggestion
  const handleCreateOrEdit = (values: `typeof form data`): void => {
```

Reply:
The other usage of `_formValues` is in mobile folder

Reply:
As for mobile - yes, in progress. Actually, mobile PR was a trigger of finding this approach in frontend